### PR TITLE
 [dhd] allow People application to save contacts, let Phone application access contacts, call list

### PR DESCRIPTION
**Pull request by Nokius**
This sed command from @mlehtima fixes the issue on Sonys scorpion, that /tmp has the wrong permissions. Which had effects in the peoples and phone applications.

[source of the sed command ](http://www.merproject.org/logs/%23sailfishos-porters/%23sailfishos-porters.2016-10-01.log.html#t2016-10-01T16:52:18)

### CODE REVIEW

**Comment by sledges** [comment]
@Nokius , thanks for the PR!

While it's being tested on Jolla devices, please rename the commit message not to say what it does (it's obvious from the diff), but why it does it. And please use the shorter tag [dhd] as per looking at commit history

**Comment by Nokius** [comment]
Sorry for the typo mess

File: droid-hal-device.inc [hunk 1]
```diff
@@ -306,7 +306,7 @@ FSTAB_FILES="$(echo %{android_root}/out/target/product/%{device}/root/fstab.* %{
 shopt -u nullglob
 
 
-%{dhd_path}/helpers/makefstab --files $FSTAB_FILES  --skip auto /acct /boot /cache /data /misc /recovery /staging /storage/usbdisk /sys/fs/cgroup /sys/fs/cgroup/memory /sys/kernel/debug  /sys/kernel/config /dev/usb-ffs/adb %{?makefstab_skip_entries} --outputdir tmp/units
+%{dhd_path}/helpers/makefstab --files $FSTAB_FILES  --skip auto /acct /boot /cache /data /dev/usb-ffs/adb /misc /recovery /staging /storage/usbdisk /sys/fs/cgroup /sys/fs/cgroup/memory /sys/kernel/debug  /sys/kernel/config /tmp %{?makefstab_skip_entries} --outputdir tmp/units
```
Comment (sledges) [inline droid-hal-device.inc]: Could you move `/dev/usb-ffs/adb` to where it was originally, so the diff doesn't look confusing?

Comment (Nokius) [inline droid-hal-device.inc]: sure I can, the change was done to have all path in alphabetic order. It looked to me it was it once :-s

Comment (sledges) [inline droid-hal-device.inc]: Then that would deserve a separate commit (1 logical change per commit :wink:)
